lesson/mineral_counter/.env

```diff
@@ -0,0 +1,2 @@
+NUTRITIONIX_API_KEY=c79e58ed2b41b51fa5e719916042b9da
+NUTRITIONIX_APP_ID=5876afdb
```

Comment:
Don't commit secrets. You'll need to rewrite the Git history to remove the commit that added these.